docs/next/guides/internationalization/layout-direction.md

```diff
@@ -0,0 +1,221 @@
+---
+title: Layout direction
+metaTitle: Layout direction - Guide - Handsontable Documentation
+permalink: /next/layout-direction
+canonicalUrl: /layout-direction
+tags:
+  - rtl
+  - right-to-left
+  - arabic
+  - hebrew
+  - persian
+  - farsi
+  - internationalization
+  - localization
+  - L10n
+  - i18n
+---
+
+# Layout direction
+
+[[toc]]
+
+Configure Handsontable's layout direction, to properly handle right-to-left (RTL) and left-to-right (LTR) [languages](@/guides/internationalization/language.md).
+
+## About layout direction
+
+To properly display Handsontable's UI and data in LTR languages (such as English, Chinese, or Russian) and RTL languages (such as Arabic, Persian, or Hebrew), 
```

Comment:
@warpech - are we sure Chinese is a good example here?
- The term **Chinese** is not precise:
    - It's not a language, but a group of languages
    - It can mean either Mandarin or Cantonese (the latter used in Hong Kong, so probably significant)
    - It can mean Simplified vs Traditional
- I don't think any "variant" of Chinese can be described as an LTR language, I'd guess that on paper they're top-to-bottom, and on the web they're **usually** LTR (but not necessarily)